docs/http_api.rst

```diff
@@ -252,3 +272,62 @@ Sent later when (or if) delivery reports are received:
 * ``delivery_succeeded``: provider confirmed that the message was delivered.
 * ``delivery_failed``: provider declared that message delivery failed.
 * ``delivery_pending``: provider is still attempting to deliver the message.
+
+
+.. _status-events:
+
+Status events
+-------------
+
+Status events ``POST``\ed to the ``status_url`` specified in :http:post:`/channels/` have the following format:
+
+.. http:post:: /status/url
+
+   :param str component:
+      The :ref:`component <status-components>`  relevant to this status event.
+   :param str status:
+      The :ref:`status level <status-levels>` this event was categorised under.
+   :param list reasons:
+      An array of human-readable strings associated with this status event.
+   :param dict details:
+      Details specific to this event intended to be used for debugging
+      purposes. For example, if the event was related to a component
+      establishing a connection, the host and port are possible fields.
+
+
+**Request Example**:
+
+.. sourcecode:: json
+
+   {
+      component: 'smpp',
```

Comment:
Good point, will add the channel id to be consistent with the events we send to `event_url`.